postreise/analyze/generation/summarize.py

```diff
@@ -44,8 +44,8 @@ def sum_generation_by_type_zone(
 
 
 def sum_generation_by_state(scenario: Scenario) -> pd.DataFrame:
-    """Get the generation of each resource from the scenario by state, including
-    totals for the given interconnects and for all states.
+    """Get the total generation for each generator type and state combination, adding
+      totals for the interconnects and for all states.
```

Comment:
Should we have an indent here or it generates automatically?

Reply:
Good catch

Reply:
Done